app/controllers/v3/processes_controller.rb

```diff
@@ -119,6 +119,7 @@ def find_process_and_space
 
   def ensure_can_write
     unauthorized! unless permission_queryer.can_manage_apps_in_space?(@space.guid)
+    suspended! unless permission_queryer.is_space_org_active?(@space.guid)
```

Comment:
Add tests to `spec/request/processes_spec.rb`:
- PATCH /v3/processes/:guid
- POST /v3/processes/:guid/actions/scale
- DELETE /v3/processes/:guid/instances/:index

Reply:
POST /v3/processes/:guid/actions/scale not yet